packages/test/test-drivers/src/routerliciousTestDriver.ts

```diff
@@ -44,6 +46,7 @@ function getConfig(
 		// The deltaStorageUrl is firstly set to https://dummy-historian to make the workflow successful.
 		return {
 			serviceEndpoint: {
+				deltaStreamUrl: "",
```

Comment:
Lines 60-64 below feel like also need a change, for the case where there is no discoveryEndpoint.  Which makes me realize, we probably started using a new subdomain for nexus? If so, we'll also want to add new DNS entries for the FF test cluster, I can do that once we confirm if it's necessary.